grammar.js

```diff
@@ -167,11 +167,13 @@ module.exports = grammar({
 
     import_header: $ => seq(
       "import",
-      $.identifier,
-      optional(choice(seq(".*"), $.import_alias)),
```

Comment:
Note that this construct (single child `seq` rule) is currently broken on tree-sitter master, so this (or a similar) change is needed to keep being able to generate a compatible parser.